allocation.md

```diff
@@ -1,53 +1,406 @@
+    "controllerKind": "deployment",
+    "label": {
+      "app": "cost-analyzer",
+      "app_kubernetes_io_instance": "kubecost",
+      "app_kubernetes_io_name": "cost-analyzer",
+      "name": "kubecost",
+      "pod_template_hash": "94dc86fc"
+    },
+    "namespace": "kubecost",
+    "node": "gke-niko-pool-2-9182dfa7-okb2",
+    "pod": "kubecost-cost-analyzer-94dc86fc-lwvrm",
+    "service": [
+      "kubecost-frontend",
+      "kubecost-cost-analyzer"
+    ]
+  },
+  "window": {
+    "start": "2021-03-11T00:00:00-07:00",
+    "end": "2021-03-12T00:00:00-07:00"
+  },
+  "start": "2021-03-11T07:00:00Z",
+  "end": "2021-03-11T17:00:00Z",
+  "minutes": 600,
+  "cpuCores": 0.200399,
+  "cpuCoreRequestAverage": 0.2,
+  "cpuCoreUsageAverage": 0.004317,
+  "cpuCoreHours": 2.00399,
+  "cpuCost": 0.044344,
+  "cpuEfficiency": 0.021583,
+  "gpuHours": 0,
+  "gpuCost": 0,
+  "networkCost": 1.3e-05,
+  "pvBytes": 11453246122.666668,
+  "pvByteHours": 114532461226.66667,
+  "pvCost": 0.005845,
+  "ramBytes": 59791087.387687,
+  "ramByteRequestAverage": 57671680,
+  "ramByteUsageAverage": 52687168.319468,
+  "ramByteHours": 597910873.876871,
+  "ramCost": 0.001652,
+  "ramEfficiency": 0.913571,
+  "sharedCost": 0,
+  "externalCost": 0,
+  "totalCost": 0.051853,
+  "totalEfficiency": 0.053611
+}
+```
+
+### Special types of allocation
+- `__idle__` refers to resources on a cluster that were not dedicated to a Kubernetes object. An idle resource can be shared (proportionally or evenly) with the other allocations from the same cluster. (See the argument `shareIdle`.)
+- `__unallocated__` refers to aggregated allocations without the selected `aggregate` field; e.g. aggregating by `label:app` might produce an `__unallocated__` allocation composed of allocation without the `app` label.
+- `__unmounted__` (or "Unmounted PVs") refers to the resources used by PersistentVolumes that aren't mounted to a Pod using a PVC, and thus cannot be allocated to a Pod.
+
+## Querying
+
+```
+GET /model/allocation
+```
+Argument | Default | Description
+--: | :--: | :--
+window (required) | — | Duration of time over which to query. Accepts: words like `today`, `week`, `month`, `yesterday`, `lastweek`, `lastmonth`; durations like `30m`, `12h`, `7d`; RFC3339 date pairs like `2021-01-02T15:04:05Z,2021-02-02T15:04:05Z`; unix timestamps like `1578002645,1580681045`.
+aggregate | | Field by which to aggregate the results. Accepts: `cluster`, `namespace`, `controllerKind`, `controller`, `service`, `label:<name>`, and `annotation:<name>`. Also accepts comma-separated lists for multi-aggregation, like `namespace,label:app`.
+accumulate | false | If `true`, sum the entire range of sets into a single set.
+idle | true | If `true`, include idle cost (i.e. the cost of the un-allocated assets) as its own allocation. (See [special types of allocation](#special-types-of-allocation).)
+external | false | If `true`, include external costs in each allocation.
```

Comment:
```suggestion
external | false | If `true`, include [external costs](http://docs.kubecost.com/getting-started#out-of-cluster) in each allocation.
```

Is there a better link for OOC costs explanation than the install instructions?

Reply:
Not to my knowledge. Is this where you would send people, @dwbrown2?

Reply:
Tagging @dwbrown2 again but I don't think we have to wait if he's busy. This link seems reasonable to me.